tests/system/python/e2e/test_e2e_modbus_c_pi.py

```diff
@@ -93,13 +93,13 @@ def start_south_north(self, reset_and_start_fledge, add_south, remove_directorie
         add_south(SOUTH_PLUGIN, south_branch, fledge_url, service_name=SVC_NAME, config=cfg,
                   plugin_lang="C", start_service=False, plugin_discovery_name=PLUGIN_NAME)
 
-        start_north_pi_server_c(fledge_url, pi_host, pi_port, pi_token)
+        start_north_pi_server_c_web_api(fledge_url, pi_host, pi_port, pi_db=pi_db, pi_user=pi_admin, pi_pwd=pi_passwd)
```

Comment:
This change alone is not sufficient, as the tests rely on dependent values further downstream, which may still cause issues.

And related Job run says FAILED in console though XML reports is missing from the build run